src/record_signal.cc

```diff
@@ -411,6 +411,29 @@ static void handle_desched_event(RecordTask* t) {
    * That may be a kernel bug, but we handle it by just
    * continuing until we we continue past the arm-desched
    * syscall *and* stop seeing signals. */
+
+  const auto untraced_record_only_exit =
+    uintptr_t(RR_PAGE_SYSCALL_UNTRACED_RECORDING_ONLY) + 4;
```

Comment:
This +4 is ARM-specific right? Fix this to be correct for all architectures?

Reply:
Ah you are right…. Apparently I’m too used to fixed instruction with now…

Reply:
On x86 we usually can't have nice things ... but at least the syscall instructions all have the same size!

Reply:
actually you can probably have variable length syscall instructions with prefixes. I guess that would still "work".